src/controllers/profile.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { Context } from "koa";
+import { MiscService } from "../services/misc.service";
+import { UserTransfromServiceResponse } from "../types/index";
+import profile from "../services/profile.service";
+export default class ProfileController {
+  public static async profile(ctx: Context) {
+    let requestMetadata = MiscService.getRequestMetadata(ctx);
+    const format = ctx.request.query.format as string;
+    const { credBucketDetails } = ctx.request.body as any;
+    const response = await profile(credBucketDetails, format);
+    ctx.body = response.body;
+    MiscService.transformerPostProcessor(ctx, response.status);
```

Comment:
transformerPostProcessor-> postProcess or something in that direction function names should action/verbs

Reply:
This is addressed